plugins/mimecast/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM --platform=linux/amd64 rapid7/insightconnect-python-3-slim-plugin:5.4.9
+FROM --platform=linux/amd64 rapid7/insightconnect-python-3-slim-plugin:5.5.1
```

Comment:
This will need updating to 5.5.2

Reply:
this has been updated to use 5.5.3 after new versions of SDK has been released